google/cloud/storage/internal/signed_url_requests.h

```diff
@@ -19,6 +19,7 @@
 #include "google/cloud/storage/internal/generic_request.h"
 #include "google/cloud/storage/signed_url_options.h"
 #include "google/cloud/storage/well_known_parameters.h"
+#include <cctype>
```

Comment:
Why here rather than the cc file?

Reply:
Fixed, I forgot to move it when I moved the function.